src/frontend/src/flows/faq/index.ts

```diff
@@ -8,69 +9,91 @@ export { questions } from "./questions";
 
 // The rendered (list item) question
 function renderQuestion(faq: Question) {
-  return html`<li id=${faq.anchor}>
-    <h3>${faq.question}</h3>
-    <p>${faq.answer}</p>
-    <ul class="links-list">
-      ${Object.values(faq.links).map(
-        (link) =>
-          html`<li>
-            &middot;
-            <a
-              class="textLink"
-              rel="noopener noreferrer"
-              href="${link.link}"
-              target="_blank"
-              >${link.name}</a
-            >
-          </li>`
-      )}
-    </ul>
+  return html`<li
+    class="faq__question"
+  >
+    <details
+    id=${faq.anchor} >
+    <summary class="faq__question-summary">
+      ${faq.question}
+    <div class="faq__question-underline"></div>
+    </summary>
+    <div>
+      <p class="faq__answer">${faq.answer}</p>
+      <ul class="faq__answer-links">
+        ${Object.values(faq.links).map(
```

Comment:
Again I think we probably want to control the order in which these links appear.

Reply:
Agreed, will change.

Reply:
Done!